app/src/main/java/mobi/visamate/datasources/network/ContributorsNetwork.kt

```diff
@@ -0,0 +1,32 @@
+package mobi.visamate.datasources.mock
+
+import mobi.visamate.datasources.network.ContributorsRetrofitInterface
+import mobi.visamate.model.Contributor
+import retrofit2.Callback
+import retrofit2.Converter
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+/**
+ * Created by gabriel on 21/2/18.
+ */
+class ContributorsNetwork {
+
+    companion object {
+
+        private var gsonConverterFactory: Converter.Factory = GsonConverterFactory.create()
+
+        fun getContributorsFromNetwork(callback: Callback<List<Contributor>>) {
+
+            val retrofit = Retrofit.Builder()
```

Comment:
I reckon we could inject retrofit instead of building up here.

`Network` classes could only enqueue calls, so we could use these classes to encapsulate each domain within our app, such as Contributors as used here.

This is to avoid building many retrofit instances for each `Network` class.

Reply:
Yes, thats a good idea, can you please add Dagger 2 on the next feature?